resources/definitions/fdmprinter.def.json

```diff
@@ -3191,6 +3191,32 @@
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
+                "cool_fan_min_at_height":
+                {
+                    "label": "Initial Fan Speed at Height",
+                    "description": "The height at which the fans spin at Initial Fan Speed. The fans are off below this height.",
```

Comment:
What happens when cool_fan_full_layer equals cool_fan_min_layer??
Please add this to the description.